modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessor.java

```diff
@@ -230,19 +238,57 @@ protected void doInitialise() throws InitialisationException {
 
   private ComponentExecutor<T> createComponentExecutor() {
     Map<String, Object> params = new HashMap<>();
-    componentModel.getAllParameterModels().stream()
-        .filter(p -> p.getModelProperty(FieldOperationParameterModelProperty.class).isPresent())
-        .forEach(p -> {
-          ValueResolver<?> resolver = resolverSet.getResolvers().get(p.getName());
+
+    final ValueResolvingContext resolvingContext = ValueResolvingContext.from(getInitialiserEvent());
+
+    componentModel.getParameterGroupModels().stream().forEach(group -> {
+      if (group.getName().equals(DEFAULT_GROUP_NAME)) {
+        group.getParameterModels().stream()
+            .filter(p -> p.getModelProperty(FieldOperationParameterModelProperty.class).isPresent())
+            .forEach(p -> {
+              ValueResolver<?> resolver = resolverSet.getResolvers().get(p.getName());
+              if (resolver != null) {
+                try {
+                  params.put(getMemberName(p), resolveValue(resolver, resolvingContext));
+                } catch (MuleException e) {
+                  throw new MuleRuntimeException(e);
+                }
+              }
+            });
+      } else {
+        ParameterGroupDescriptor groupDescriptor = group.getModelProperty(ParameterGroupModelProperty.class)
```

Comment:
This whole else could be refactored into several methods, it's a little tricky understand what is doing.